docs/app/Examples/views/Comment/Content/CommentExampleReplyForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Button, Comment, Form, Icon } from 'stardust'
+
+const CommentExampleReplyForm = () => (
+  <Comment.Group>
+    <Comment>
+      <Comment.Avatar as='a' src='http://semantic-ui.com/images/avatar/small/steve.jpg' />
+      <Comment.Content>
+        <Comment.Author as='a'>Steve Jobes</Comment.Author>
+        <Comment.Metadata>
+          <div>2 days ago</div>
+        </Comment.Metadata>
+        <Comment.Text>Revolutionary!</Comment.Text>
+        <Comment.Actions>
+          <Comment.Action active>Reply</Comment.Action>
+        </Comment.Actions>
+        <Comment.ReplyForm>
```

Comment:
We have same situation with `List.Icon`, if we add `reply` prop on `Form`, seems we'll need to remove `List.Icon`.

@jcarbo ?